docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1473,6 +1473,7 @@ Converts a JSON string into a structured JSON object.
 | Name           | Required  | Default  | Description
 | `field`        | yes       | -        | The field to be parsed
 | `target_field` | no        | `field`  | The field to insert the converted structured object into
+| `add_to_root`  | no        | false    | Flag that forces the serialized json to be injected into the top level of the document. When set to `true`, `target_field` will be ignored.
```

Comment:
I think the name is ok, we refer to the top level of a json document as root in other places too.